### ISSUE TITLE
502 error while using the python API

### ISSUE THREAD

**Issue by sarim-zafar** [truthy/botornot-python#4]
raise HTTPError(http_error_msg, response=self)
HTTPError: 502 Server Error: Bad Gateway for url: http://truthy.indiana.edu/botornot/api/1/check_account

**Comment by clayadavis** [comment]
This happens sometimes when the server has issues, usually over-capacity, so it's not really an issue with this piece of code. I think the solution is a troubleshooting/FAQ doc item, not a code fix. I'm going to make a new issue so as not to take yours over.

**Comment by clayadavis** [comment]
Additional info available in the [Wiki Troubleshooting & FAQ](/truthy/botornot-python/wiki/Troubleshooting-&-FAQ#502-bad-gateway).

**Comment by kurtawirth** [comment]
I'm having this same error, despite requesting from the U.S. Any ideas?

**Comment by clayadavis** [comment]
@kurtawirth how long have you been getting that error? And do you get it every time you send a request or just some of the time? As mentioned above and in the FAQ, it's usually a server or connectivity error and resolves itself after a little while, but I can investigate further if you've been seeing it for more than a couple hours, say.

**Comment by kurtawirth** [comment]
I've gotten it off and on for the past few days. Let me try again this
weekend and try to see what I can do on my end and get back to you!

On Sep 21, 2018 6:34 PM, "Clayton A Davis" <notifications@github.com> wrote:

@kurtawirth <https://github.com/kurtawirth> how long have you been getting
that error? And do you get it every time you send a request or just some of
the time? As mentioned above and in the FAQ, it's usually a server or
connectivity error and resolves itself after a little while, but I can
investigate further if you've been seeing it for more than a couple hours,
say.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/IUNetSci/botometer-python/issues/4#issuecomment-423688550>,
or mute the thread

**Comment by clayadavis** [comment]
OK that helps, it means it's probably not you or your code, but some other server in the chain. Would you mind pasting the exact URL in the 502 you're getting? That would help me narrow it down.

**Comment by kurtawirth** [comment]
![image](https://user-images.githubusercontent.com/35779108/45913472-00a72980-be01-11e8-8916-bdacd3e95d4c.png)

Here's the full set of error code. Note that there are two; I'll look through the issues to find one on the "NoTimelineError".
